css-two-column-layout/styles.css

```diff
@@ -0,0 +1,95 @@
+body {
+  font-family: Karla, sans-serif;
+}
+
+.container {
+  width: 80%;
+  margin: auto;
+  max-width: 800px;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
```

Comment:
The only styling on the `row` class should be `display: flex`. You should move these to a separate class you can add in addition to the `row` class.